src/deprecated/index.ts

```diff
@@ -0,0 +1,30 @@
+/** This is the place where we keep components that are deprecated.
+ *  We don't recommend using it in production.
+ *  If you already use them, please move to the suggested alternative components
+ *
+ *  But, they are published on npm and you can import them.
+ *  example: import {FormGroup} from '@primer/react/deprecated
+ */
+
+export {default as BorderBox} from '../BorderBox'
```

Comment:
Are we intentionally importing these from `src` instead of moving them to `src/deprecated/`?

Reply:
Well we don't have all the draft component in a separate folder, so I kept it the same for deprecated